CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [11.4.0](https://github.com/paritytech/substrate-api-sidecar/compare/v11.3.19..v11.4.0) (2022-05-18)
+
+**Upgrade Priority** Low
+
+### Features
+
+- feat(balance-info): add query param to convert free balance to human ([#914](https://github.com/paritytech/substrate-api-sidecar/pull/914)) ([f1e03d6](https://github.com/paritytech/substrate-api-sidecar/commit/f1e03d6c771d74cd4848975c2ebdf42bd9ab8d8b))
```

Comment:
convert to human?

Reply:
the PR description was too long `Returns the any atomic balance as a human readable number based on the given chains token decimal`

Reply:
I simply copied the commit name as it was found here https://github.com/paritytech/substrate-api-sidecar/pull/914 should I add "readable format" to the end so it reads, "add query param to convert free balance to human readable format"?

Reply:
That makes sense but as the `CHANGELOG` is used for humans we should aim to a descriptive as possible.
We should do a better job with better descriptions in the future, sorry

Reply:
Ok updated the Change log to reflect this feedback. Thanks folks